src/embeds/components/Items/index.js

```diff
@@ -65,6 +65,16 @@ export default class ItemsEmbed extends Component {
   render () {
     const { ids, statIds, items, itemstats, className } = this.props;
 
+    // Do not render until all items are loaded.
+    const existingItems = Object.keys(items);
+    const itemsExists = ids.reduce((exists, curId) =>
+      exists && (existingItems.indexOf(curId) > -1),
+    true);
+    if (!itemsExists) {
```

Comment:
hmm what was the error? it was made to be able to display item boxes even if the data hadn't loaded yet.

ideally we keep that somehow, because this will just display nothing until the data is loaded instead of `n` placeholder boxes